react/features/prejoin/functions.js

```diff
@@ -57,6 +58,50 @@ export function getActiveVideoTrack(state: Object): Object {
     return null;
 }
 
+/**
+ * Executes the dial out request.
+ *
+ * @param {string} url - The url for dialing out.
+ * @param {Object} body - The body of the request.
+ * @param {string} reqId - The unique request id.
+ * @returns {Object}
+ */
+export async function executeDialOutRequest(url: string, body: Object, reqId: string) {
```

Comment:
These should probably be moved to the invite feature and used there to avoid duplicating the code.